libs/openFrameworksCompiled/project/android/build.gradle

```diff
@@ -81,7 +96,12 @@ task ofNdkSetup {
  * Helper function to run make with a given set of options
  */
 def ofRunMake(List opts) {
-    List cmd = [rootProject.ofNdkSetup.ext.make, "-j4"] + opts
+    if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
+        // Enable parallel make on all platforms except Windows.
+        // MinGW (at time of writing, 2015-02-16) has issues running parallel make.
```

Comment:
is there a MinGW bug report for this? if yes, would be great to link it here, so we can check if this is still current in the future.